### PR TITLE
CGE-193 fix bug when it is possible to enter more than 129 character on E-mail fields when changing one

### PR DESCRIPTION
**Pull request by svyatoslav145**

### CODE REVIEW

File: src/main/java/com/hillel/items_exchange/controller/UserController.java [hunk 1]
```diff
@@ -101,8 +102,8 @@ public String updateUserPassword(@Valid @RequestBody UserChangePasswordDto userC
             @ApiResponse(code = 403, message = "FORBIDDEN"),
             @ApiResponse(code = 409, message = "CONFLICT")})
     @ResponseStatus(HttpStatus.ACCEPTED)
-    public String updateUserEmail(@Valid @RequestBody UserChangeEmailDto userChangeEmailDto, Principal principal)
-            throws DataConflictException {
+    public String updateUserEmail(@Valid @RequestBody UserChangeEmailDto userChangeEmailDto, BindingResult bindingResult,
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/controller/UserController.java:105]: Could you explain why do we need BindingResult here?

File: src/main/java/com/hillel/items_exchange/dto/UserChangeEmailDto.java [hunk 2]
```diff
@@ -18,10 +19,10 @@
 public class UserChangeEmailDto {
 
     @NotEmpty(message = "{invalid.not-empty}")
+    @Size(max = 129, message = "{too.big.email}")
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/dto/UserChangeEmailDto.java:22]: If I remember correctly we already discussed it and desided just to use Email annotation? @Jack11M , @drw85 do you remember such discussion?

File: src/main/java/com/hillel/items_exchange/controller/UserController.java (hunk 1 above)
Comment (svyatoslav145) [reply src/main/java/com/hillel/items_exchange/controller/UserController.java:105]: I'd like to discuss this.

File: src/main/java/com/hillel/items_exchange/dto/UserChangeEmailDto.java (hunk 2 above)
Comment (svyatoslav145) [reply src/main/java/com/hillel/items_exchange/dto/UserChangeEmailDto.java:22]: I'd like to discuss this.

**Pull request closed without merging**